gradle/libs.versions.toml

```diff
@@ -45,6 +45,7 @@ postgresql = "42.6.0"
 immutables-value = "2.10.0"
 selenium = "3.141.59"
 rauschig = "1.2.0"
+kyuubi = "1.8.0"
```

Comment:
Kyuubi 1.8.1 is under RC[1] period, which provides Scala 2.13 jars in the Maven repo, you may want to have a try with the staging repo[2].

[1] https://www.mail-archive.com/dev@kyuubi.apache.org/msg03096.html
[2] https://repository.apache.org/content/repositories/orgapachekyuubi-1042/

Reply:
got it, thanks!